src/elements/inputGroup/InputGroup.tsx

```diff
@@ -1,16 +1,28 @@
 import React, { FC } from 'react';
 import classnames from 'classnames';
-import Props from './InputGroup.types';
+import InputLabel from '../inputLabel';
 import Icon from '../icon';
+import Props from './InputGroup.types';
 
 const InputGroup: FC<Props> = ({
+  value,
+  id,
+  className,
+  inputLabelProps,
+  dataTestId,
+  maxLength,
   prepend,
   append,
-  inputPlaceholder,
+  placeholder,
   disabled,
   error,
+  size = 'lg',
+  onChange,
+  onFocus,
+  onBlur,
 }: Props): JSX.Element => (
-  <div>
+  <div className={classnames(`size-${size}`, 'form-group')}>
```

Comment:
The prop "className" that the component receives is usually passed to the first div of the component, so we can customize the style of the entire component and not only the style of the input. Can you move it up here?

Reply:
Yes, you are right. Done :+1: